src/__tests__/ParseUser-test.js

```diff
@@ -229,6 +229,39 @@ describe('ParseUser', () => {
     });
   });
 
+  it('can log in as a user with POST method', (done) => {
```

Comment:
Can you add an integration test?

Reply:
I did, and it passed. I believe that the server is reading the credentials from the body as the short-circuit for query params is falsy though I could be wrong about that.